### ISSUE TITLE
Markdown syntax highlight breaks after '>' blockquote sign inside fenced code block

### ISSUE THREAD

**Issue by fdefreitas** [Microsoft/vscode#12379]
- VSCode Version: Code 1.5.2 (66f37fd2a99eb9d628dd374d81d78835b410c39b, 2016-09-12T13:05:12.065Z)
- OS Version: Darwin x64 15.6.0

Steps to Reproduce:
1. Create a fenced code block that includes '>' in it. Example 

```
class NewComponentController {
    static $inject: Array<string> = [...];
}
```
1. The highlighting breaks even after the end of the fenced code block. but it works okay in the preview and any other markdown editors.

<img width="1403" alt="screen shot 2016-09-21 at 13 25 38" src="https://cloud.githubusercontent.com/assets/1840544/18710944/9d233fca-7fff-11e6-8f99-d23f120b7ae2.png">

**Comment by roefunc** [comment]
Same here in pure Javascript with Flow type annotations:

![screen shot 2016-10-19 at 12 25 01](https://cloud.githubusercontent.com/assets/16400678/19517097/1b92bca2-95f7-11e6-8fcb-389e4735f8de.png)

**Comment by roefunc** [comment]
Just a quick note that this makes syntax highlighting pretty useless for React Native projects :-(

**Comment by mjbvz** [comment]
This was fixed by #14626. 

![screen shot 2016-11-17 at 3 49 23 pm](https://cloud.githubusercontent.com/assets/12821956/20412731/5bf885c2-acdd-11e6-97c9-3e1e0871681d.png)


Give the latest insiders builds a try and please let me know if you still see this issue.  Thanks.
